Sources/ComposableArchitecture/Instrumentation.swift

```diff
@@ -72,19 +72,30 @@ public class Instrumentation {
   /// - Parameter timing: When this callback is being invoked (pre|post)
   /// - Parameter kind: The store's activity that to which this callback relates (state update, deduplication, etc)
   public typealias Callback = (_ info: CallbackInfo<Any, Any>, _ timing: CallbackTiming, _ kind: CallbackKind) -> Void
-  let callback: Callback?
+  var callback: Callback?
```

Comment:
mind making this `private(set) var` and the same for the other one?

Reply:
[![cool](https://graphite-user-uploaded-assets.s3.amazonaws.com/default-memes/cool.gif)](https://app.graphite.dev/macros/thebrowsercompany)